test/unit/cart.js

```diff
@@ -132,9 +132,7 @@ describe('Cart class', () => {
         fetchCart.restore();
       });
     });
-  });
 
-  describe('fetchData()', () => {
```

Comment:
why do we lose this describe?

Reply:
This is a duplicate , I think I mistakenly added it in one of the previous PRs, these tests are still under the `fetchData()`